main_routers/characters_router.py

```
@@ -0,0 +1,706 @@
+    new_voice_id = characters['猫娘'][name].get('voice_id', '')
+    voice_id_changed = (old_voice_id != new_voice_id)
+    is_current_catgirl = (name == characters.get('当前猫娘', ''))
+    session_ended = False
+    
+    if voice_id_changed and is_current_catgirl and name in session_manager:
+        if session_manager[name].is_active:
+            if session_manager[name].websocket:
+                try:
+                    await session_manager[name].websocket.send_text(json.dumps({
+                        "type": "reload_page",
+                        "message": "语音已更新，页面即将刷新"
```

Ruff (0.14.8)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)